src/integrationTest/java/uk/gov/hmcts/sptribs/controllers/CaseManagementControllerIT.java

```diff
@@ -0,0 +1,426 @@
+
+        mockMvc.perform(post("/case/dss-orchestration/create")
+            .contentType(APPLICATION_JSON)
+            .header(SERVICE_AUTHORIZATION, TEST_AUTHORIZATION_TOKEN)
+            .header(AUTHORIZATION, TEST_AUTHORIZATION_TOKEN)
+            .content(objectMapper.writeValueAsString(dssCaseData))
+            .accept(APPLICATION_JSON))
+            .andExpect(status().is5xxServerError())
+            .andExpect(result -> assertTrue(result.getResolvedException() instanceof CaseCreateOrUpdateException));
+    }
+
+    @Test
+    void shouldSuccessfullyUpdateCaseWhenUpdateCaseControllerTriggered() throws Exception {
+        final DssCaseData dssCaseData = DssCaseData.builder()
+            .subjectFullName("Test Name")
+            .subjectDateOfBirth(LocalDate.of(1990, 1, 1))
+            .subjectEmailAddress("test@email.com")
+            .subjectContactNumber("07123412345")
+            .caseTypeOfApplication("CIC")
+            .build();
+
+        final CaseData caseData = CaseData.builder()
+            .dssCaseData(dssCaseData)
+            .build();
+
+        final uk.gov.hmcts.ccd.sdk.api.CaseDetails<CaseData, State> caseDetails =
+            new uk.gov.hmcts.ccd.sdk.api.CaseDetails<>();
+        caseDetails.setId(TEST_CASE_ID);
+        caseDetails.setData(caseData);
+
+        final uk.gov.hmcts.reform.ccd.client.model.CaseDetails reformCaseDetails =
+            objectMapper.convertValue(caseDetails, new TypeReference<>() {});
+
+        final StartEventResponse startEventResponse = StartEventResponse.builder()
+            .eventId(CITIZEN_CIC_CREATE_CASE)
+            .token("startEventToken")
+            .caseDetails(reformCaseDetails)
+            .build();
+
+        final CaseDataContent caseDataContent = CaseDataContent.builder()
+            .data(caseData)
+            .event(uk.gov.hmcts.reform.ccd.client.model.Event.builder()
+                .id(CITIZEN_CIC_UPDATE_CASE)
+                .build()
+            )
+            .eventToken("startEventToken")
+            .build();
+
+        stubForIdamDetails(TEST_AUTHORIZATION_TOKEN, CASEWORKER_USER_ID, ST_CIC_CASEWORKER);
+        when(authTokenGenerator.generate()).thenReturn(S2S_TOKEN);
+        when(coreCaseDataApi.startEventForCitizen(
+            AUTH_TOKEN,
+            S2S_TOKEN,
+            "2",
+            ST_CIC_JURISDICTION,
+            ST_CIC_CASE_TYPE,
+            String.valueOf(TEST_CASE_ID),
+            CITIZEN_CIC_UPDATE_CASE
+        )).thenReturn(startEventResponse);
+        when(coreCaseDataApi.submitEventForCitizen(
+            AUTH_TOKEN,
+            S2S_TOKEN,
+            "2",
+            ST_CIC_JURISDICTION,
+            ST_CIC_CASE_TYPE,
+            String.valueOf(TEST_CASE_ID),
+            true,
+            caseDataContent
+        )).thenReturn(reformCaseDetails);
+
+        final String response = mockMvc.perform(put("/case/dss-orchestration/1616591401473378/update?event=UPDATE")
```

Comment:
Can you assign /case/dss-orchestration/1616591401473378/update?event=UPDATE url to a constant